operations/helm/charts/alloy/Chart.yaml

```diff
@@ -2,8 +2,9 @@ apiVersion: v2
 name: alloy
 description: 'Grafana Alloy'
 type: application
-version: 0.1.0
+version: 0.1.1
 appVersion: 'v1.0.0'
+icon: https://raw.githubusercontent.com/grafana/alloy/main/docs/sources/assets/alloy_icon_orange.svg
```

Comment:
(This link won't work until the PR is merged)

Reply:
You can verify that it will work by `s/main/helm-chart-icon` though: https://raw.githubusercontent.com/grafana/alloy/helm-chart-icon/docs/sources/assets/alloy_icon_orange.svg